src/main/proto/netflix/titus/agent.proto

```diff
@@ -284,6 +284,9 @@ message ContainerInfo {
 
     // Used internally by the agent to store information about a running container
     optional RunningContainerInfo runState = 39;
+
+    /// (Optional) Amount of shared memory to allocate (must be < memoryMB)
```

Comment:
`<` or `<=`?

Netflix/titus-api-definitions#93 says `<=`, we should probably align them both.

Reply:
Good catch = `<=`